Makefile

```diff
@@ -11,6 +11,9 @@ build: fioctl-linux-amd64 fioctl-linux-arm64 fioctl-windows-amd64 fioctl-darwin-
 fioctl-static:
 	CGO_ENABLED=0 go build -a -ldflags '-w -extldflags "-static"' -o ./bin/fioctl-static ./main.go
 
+fioctl-cgo-pkcs11:
+	CGO_ENABLED=1 go build -tags cgopki $(LDFLAGS) -o bin/$@ ./main.go
+
```

Comment:
See the test results here: https://github.com/foundriesio/fioctl/actions/runs/6355711636/job/17264165068?pr=278

And also my latest global comment which describes the rationale to keep all 3 implementations.
Just IMHO, the CGo based version is the easiest to maintain and understand.
So it could be used as a proof-of-correctness when we develop new features and tests are being added.

Reply:
We decided to keep it on the standup.

A rationale is this:
- Bash implementation is the most tested one, so it stays as a backup for a few months, before we scrap it.
- Golang + pkcs11-tool is our new default implementation, so it stays.
- Golang + ThalesIgnite is the easiest to prototype, so it stays to cross-verify the pkcs11-tool one.